FirebasePerformance/Tests/TestApp/Podfile

```diff
@@ -5,6 +5,8 @@ platform :ios, '11.0'
 #source 'sso://cpdc-eap/spec'
 #source 'https://github.com/CocoaPods/Specs.git'
 
+use_modular_headers!
```

Comment:
Curious on what's the purpose of this?

Reply:
tl;dr is that - Cocoapods way of organizing dependencies (SDKs) are different from how Swift looks at them as modules. This construct, enables a seamless compatibility of Cocoapods based dependencies with the Swift modules.

A nice read about "Modular headers" and the need for this construct is at https://blog.cocoapods.org/CocoaPods-1.5.0/.

Reply:
`use_modular_headers!` is now somewhat deprecated by `use_frameworks! :linkage => :static`. See https://blog.cocoapods.org/CocoaPods-1.9.0-beta/